### PR TITLE
first assignment changes

### PR DESCRIPTION
**Pull request by aditiastonkar**
change in header section.

### CODE REVIEW

File: Aditi/aditi_first_assignment/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<html>
+<head>
+     <title>css demo</title>
+     <link rel="stylesheet"href="style.css"/>
+</head>
+ <body>
+    <section class="header-section">
+	<header>
+	<hgroup class="website-img">
+	   <img src="fullstacklogo.PNG"alt="website logo"height=50px/>
```
Comment (sachinghuge24) [inline Aditi/aditi_first_assignment/index.html]: remove this height and add to img tag in css.

File: Aditi/aditi_first_assignment/style.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	 background-size:cover;
+	 
+	 }
+	 
+	 section.header-section{
+		 border-top:0px;
+	  background-color:white;
+	  min-height:50px;	
+	   padding-left:100px;
+	  padding-right:100px;
```
Comment (sachinghuge24) [inline Aditi/aditi_first_assignment/style.css]: remove this padding and add it to body tag.
Also add float: left to your section here.

File: Aditi/aditi_first_assignment/style.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+		text-align:center;
+		margin-top:50px;
+		margin-bottom:4px;
+		font-size:20px;
+		color:white;
+	
+	}
+	.h32{
+		text-align:center;
+		padding-top:opx
```
Comment (sachinghuge24) [inline Aditi/aditi_first_assignment/style.css]: What is opx here. add semicolun

File: Aditi/aditi_first_assignment/index.html (hunk 1 above)
Comment (aditiastonkar) [reply Aditi/aditi_first_assignment/index.html]: done

**Pull request merged**